.gitignore

```diff
@@ -4,6 +4,7 @@ _trial_temp
 apidoc
 *.pyc
 build
+.ropeproject
```

Comment:
Hey @bretttjohnson1 - rope project isn't something that is being used for this project.

Reply:
.ropeproject is some directory that got created by one of my python linters. I didn't want to accidentally commit it, so I added it in the .gitignore. I can remove it if it is a problem.

Reply:
In general, I prefer to keep references to personal developer library choices out of files committed to git (just because those are personal development environment files, unrelated to the library itself).